### PR TITLE
[FE-7750] Fix inconsistent display of mapbox logo / watermark 

### PR DESCRIPTION
**Pull request by CaitW**
# Merge Checklist
## :wrench: Issue(s) fixed:
- [ ] Author referenced issue(s) fixed by this PR:
- [ ] Fixes [FE-7750](https://jira.omnisci.com/browse/FE-7750)

## :smoking: Smoke Test
- [ ] Works in chrome
- [ ] Works in firefox
- [ ] Works in safari
- [ ] Works in ie edge
- [ ] Works in ie 11

## :ship: Merge
- [ ] author crafted PR's title into release-worthy commit message.

### CODE REVIEW

File: src/mixins/map-mixin.js [hunk 1]
```diff
@@ -277,8 +277,9 @@ export default function mapMixin(
     mapboxlogo.target = "_blank"
     mapboxlogo.innerHTML = "Mapbox"
 
-    const existingLogo = document.getElementsByClassName('mapbox-maplogo')
-    if(existingLogo.length  ) {
+    const thisMapContainer = _map._container
```
Comment (jrajav) [inline src/mixins/map-mixin.js]: How certain are we `_map` will never still be null here?

File: src/mixins/map-mixin.js [hunk 2]
```diff
@@ -277,8 +277,9 @@ export default function mapMixin(
...
+    const existingLogo = thisMapContainer.querySelectorAll('.mapbox-maplogo')
```
Comment (jrajav) [inline src/mixins/map-mixin.js]: Seems like we could easily just use `querySelector` instead here, but unlikely to matter

Comment (CaitW) [reply src/mixins/map-mixin.js]: Changed because `querySelector` is better style

File: src/mixins/map-mixin.js (hunk 1 above)
Comment (CaitW) [reply src/mixins/map-mixin.js]: Added check for null `_map`

**Pull request merged**